e2e_tests/tests/test_users.py

```diff
@@ -12,26 +12,26 @@
 from pexpect import spawn
 
 from determined.common import constants, yaml
-from determined.common.api.authentication import Authentication, Credentials, TokenStore
+from determined.common.api import authentication
 from tests import command
 from tests import config as conf
 from tests import experiment as exp
 from tests.filetree import FileTree
 
 EXPECT_TIMEOUT = 5
-ADMIN_CREDENTIALS = Credentials("admin", "")
+ADMIN_CREDENTIALS = authentication.Credentials("admin", "")
 
 
 @pytest.fixture(scope="session")  # type: ignore
-def auth() -> Authentication:
-    a = Authentication.instance()
-    a.reset_session()
-    yield a
-    TokenStore.delete_token_cache()
+def auth() -> authentication.Authentication:
+    authentication.cli_auth = None
+    yield authentication.cli_auth
```

Comment:
Does this always yield None?

Reply:
Yes it does.  I can clean that up though.

Reply:
It's better to clean that up in this PR I think. It looks weird to set it to be None, yield a None, and set it to be None again. Doesn't make straight sense to me.

Reply:
will do.